src/styles/components/_name-generator.scss

```diff
@@ -39,11 +39,24 @@
   &__add-person {
     @include svg-button('/images/add-a-person.svg');
     position: absolute;
-    bottom: 20px;
-    right: 0;
-    width: 150px;
+    bottom: 35px;
```

Comment:
Good point, this kind of thing will probably end up in the style guide, which is going to also serve as a library of styles to use in the app.